mne/inverse_sparse/mxne_inverse.py

```diff
@@ -92,6 +92,17 @@ def _prepare_gain(forward, info, noise_cov, pca, depth, loose, weights,
         raise ValueError('Depth parameter must be positive (got %s).'
                          % depth)
 
+    if len(forward['src']) == 1:
```

Comment:
Currently, discrete or mixed source spaces are not supported. I will add a check and we can include the other types of source spaces later, ok?